src/node_options.cc

```diff
@@ -16,6 +16,30 @@ using v8::Undefined;
 using v8::Value;
 
 namespace node {
+
+void PerProcessOptions::CheckOptions(std::vector<std::string>* errors) {
+  // Check any per-process options here.
+#if HAVE_OPENSSL
+  if (use_openssl_ca && use_bundled_ca) {
+    errors->push_back("either --use-openssl-ca or --use-bundled-ca can be "
+                      "used, not both");
+  }
+#endif
+  per_isolate->CheckOptions(errors);
+  per_isolate->per_env->CheckOptions(errors);
+  per_isolate->per_env->debug_options->CheckOptions(errors);
```

Comment:
Would it maybe make sense to nest these directly, like callling `debug_options->CheckOptions(errors)` from `EnvironmentOptions::CheckOptions()` and so on?

Reply:
Ah I like that, I'll give that a try. Thanks